src/test/java/pingis/services/TaskInstanceServiceTest.java

```diff
@@ -163,8 +163,10 @@ public void testUpdateTaskInstanceCode() {
 
   @Test
   public void testMarkAsDone() {
+    int oldPoints = testUser.getPoints();
     TaskInstance result = taskInstanceService.markAsDone(testTaskInstance);
     assertEquals(CodeStatus.DONE, result.getStatus());
+    assertTrue(oldPoints < testUser.getPoints());
```

Comment:
This test would be better if you calculated the expected number of points and used `assertEquals`.